ccinput/wrapper.py

```diff
@@ -432,6 +434,12 @@ def get_parser():
         default=None,
         help="Assign each atom to a fragment",
     )
+    parser.add_argument(
+        "--additional",
```

Comment:
At the moment, I don't grasp how this is different from the specifications. Could both arguments be combined?

Reply:
This additional section is designated to commands and any text in general that should not go inside blocks. The commands for this block are not checked at all and they are placed in the file after all the other blocks, but before tasks. If I implement gaussian-like specifications that we discussed in the email, this could be merged into specifications. I will probably do that.